### PR TITLE
48-enhancement

### PR DESCRIPTION
**Pull request by baolingz**
- Updated sql scripts for nysdec_solidwaste, nysdoh_healthfacilities and dsny_mtsgaragemaintenance, cause some data values have changed
- Updated wkt_geometry field for dsny_mtsgaragemaintenance, bpl_libraries at sql

### CODE REVIEW

File: sql/nysdec_solidwaste.sql [hunk 1]
```diff
@@ -30,7 +30,6 @@ SET hash =  md5(CAST((t.*)AS text)),
                     WHEN activity_desc LIKE '%RHRF%' THEN 'Recyclables Handling and Recovery'
                     WHEN activity_desc LIKE '%medical%' THEN 'Regulated Medical Waste'
                     WHEN activity_desc LIKE '%Transfer%' THEN 'Transfer Station'
-                    WHEN activity_desc LIKE '%Composting%' THEN 'Composting'
```
Comment (SPTKL) [inline sql/nysdec_solidwaste.sql]: Why is this row deleted? I checked through the dataset and there are 10 records for compose
@baolingz

**Comment by SPTKL** [comment]
@baolingz 
can you be more specific about "cause some data values have changed"

File: sql/nysdec_solidwaste.sql (hunk 1 above)
Comment (baolingz) [reply sql/nysdec_solidwaste.sql]: because there is a identical row of codes in line 28.

Comment (SPTKL) [reply sql/nysdec_solidwaste.sql]: oh ok

**Comment by baolingz** [comment]
For  nysdoh_healthfacilities, there's no  'NYC Health and Hospital Corporation' under operatorname column anymore, instead they are using 'New York City Health and Hospital Corporation'. For dsny_mtsgaragemaintenance, all the values under type column are not all letters capitalized anymore but only capitalized the first letter.

**Pull request merged**